R/pkg/R/SQLContext.R

```diff
@@ -257,23 +257,24 @@ createDataFrame.default <- function(data, schema = NULL, samplingRatio = 1.0) {
 }
 
 createDataFrame <- function(x, ...) {
-  dispatchFunc("createDataFrame(data, schema = NULL, samplingRatio = 1.0)", x, ...)
+  dispatchFunc("createDataFrame(data, schema = NULL)", x, ...)
 }
 
 #' @rdname createDataFrame
 #' @aliases createDataFrame
 #' @export
 #' @method as.DataFrame default
 #' @note as.DataFrame since 1.6.0
-as.DataFrame.default <- function(data, schema = NULL, samplingRatio = 1.0) {
```

Comment:
the `.default` methods are here for backward compatibility. 
(please see  SPARK-16693/PR #14330)
I don't think we should change the signature - or it will break any existing user. perhaps just add the @param and say "Currently not use"?